erpnext/crm/doctype/opportunity/opportunity.py

```diff
@@ -576,6 +576,38 @@ def set_missing_values(source, target):
 
 	return target_doc
 
+@frappe.whitelist()
+def make_opportunity_gate_pass(source_name, target_doc=None):
+	def set_missing_values(source, target):
```

Comment:
check_if_doc_exist when creating Vehicle Gate Pass from Opportunity just like we check in Vehicle Booking Order